src/main/java/lotto/type/SingleNumber.java

```diff
@@ -0,0 +1,83 @@
+package lotto.type;
+
+/**
+ * 로또 번호나 당첨 번호에 사용되는 번호 1개를 나타냅니다.
+ * 불변 객체입니다.
+ */
+public class SingleNumber implements Comparable<SingleNumber> {
+
+    /** 로또 번호 하한선 (경계값 포함) */
+    private static final int MIN_NUM = 1;
+
+    /** 로또 번호 상한선 (경계값 포함) */
+    private static final int MAX_NUM = 45;
+
+    private int lottoNumber;
```

Comment:
```suggestion
    private final int lottoNumber;
```

lottoNumber는 항상 있어야만 이 클래스의 의미가 살아납니다.
그런 경우엔 final을 붙여서 의미를 명확하게 표현해주세요.